defaults/main.yml

```diff
@@ -27,6 +27,7 @@ manage_firewall_rules: true
 openvpn_client_register_dns: true
 openvpn_use_crl: false
 openvpn_duplicate_cn: false
+openvpn_status_version: 2
```

Comment:
According to https://community.openvpn.net/openvpn/wiki/Openvpn24ManPage, the default is 1, not 2.
Could you change it?